### PR TITLE
mohammad/change_placeholders

### PR DESCRIPTION
**Pull request by binary-hamedanchi**
Replace `%` placeholders with `[_1]` format

### CODE REVIEW

File: src/javascript/binary/static/cashier/my_accountws.js [hunk 1]
```diff
@@ -65,8 +65,10 @@ var MyAccountWS = (function() {
         if(TUser.get().balance < 1000) {
             $(virtualTopupID + ' a')
                 .text(
-                    (text.localize('Deposit %1 virtual money into your account ') + loginid)
-                    .replace('%1', TUser.get().currency + ' 10000')
```
Comment (negar-binary) [inline src/javascript/binary/static/cashier/my_accountws.js]: let's see if any way not to hard code 10000

Comment (binary-hamedanchi) [inline src/javascript/binary/static/cashier/my_accountws.js]: Sure, I appreciate it if we could avoid these hard codes.

File: src/javascript/binary/static/cashier/my_accountws.js [hunk 2]
```diff
@@ -65,8 +66,10 @@ var MyAccountWS = (function() {
         if(TUser.get().balance < 1000) {
```
Comment (ashkanx) [inline src/javascript/binary/static/cashier/my_accountws.js]: the minimum balance shouldn't be hard-coded.

File: src/javascript/binary/pages/client/api_tokenws.js [hunk 3]
```diff
@@ -70,7 +70,7 @@ var APITokenWS = (function() {
         // Hide form if tokens count reached the maximum limit
         if(api_token.tokens.length >= maxTokens) { 
             $('#token_form').addClass(hideClass);
-            showMessage(text.localize('The maximum number of tokens (%1) has been reached.').replace('%1', maxTokens), false);
```
Comment (raunakkathuria) [inline src/javascript/binary/pages/client/api_tokenws.js]: is this tested, `text.localize` is handling it correctly?

Comment (binary-hamedanchi) [inline src/javascript/binary/pages/client/api_tokenws.js]: Yes, it is tested and works correctly.

File: src/javascript/binary/static/cashier/my_accountws.js (hunk 2 above)
Comment (binary-hamedanchi) [inline src/javascript/binary/static/cashier/my_accountws.js]: Sure, I will change it once we have required values in a WebSocket call.

**Pull request merged**